lib/ae_mdw/db/mutations/oracle_invalidation_mutation.ex

```diff
@@ -0,0 +1,36 @@
+defmodule AeMdw.Db.OracleInvalidationMutation do
+  @moduledoc """
+  Invalidates oracle tables records for a certain height.
+  """
+
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.Sync
+
+  @derive AeMdw.Db.TxnMutation
+  defstruct [:keys_delete, :records_write]
+
+  @type table_keys :: {Model.table(), [Model.key()]}
+  @type table_records :: {Model.table(), [Model.m_record()]}
+
+  @opaque t() :: %__MODULE__{
+            keys_delete: list(table_keys),
+            records_write: list(table_records)
+          }
+
+  @spec new(AeMdw.Blocks.height()) :: t()
+  def new(height) do
+    {name_dels, name_writes} = Sync.Name.invalidate(height)
```

Comment:
Why is this mutation called OracleInvalidation, but it invalidates names?

Reply:
By the previous code from the diff it should be `Sync.OracleInvalidation`.

Reply:
I don't understand what that comment means